x-pack/plugins/observability/public/components/shared/index.tsx

```diff
@@ -124,3 +124,15 @@ export function ObservabilityAlertSearchBar(props: ObservabilityAlertSearchBarPr
     </Suspense>
   );
 }
+
+const ObservabilityAlertStatusFilterLazy = lazy(
```

Comment:
nit: to make it easier to find this component, it's better to use either `AlertsStatus` (with s) or `AlertStatus` everywhere :)

Reply:
I'll remove this exports once I will finish creating the same Button group into the plugin, but thanks for noticing, I'll double check it is consistent in the whole implementation :)